enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -18,8 +27,39 @@ export const SdkContentWrapper = styled.div<{ font: string; baseUrl?: string }>`
   ${({ theme }) => getRootStyle(theme)}
 
   ${({ baseUrl }) => defaultFontFiles({ baseUrl })}
+  ${({ fontFiles }) =>
+    fontFiles?.map(
+      file => css`
+        @font-face {
+          font-family: "Custom";
```

Comment:
I asked a question about [this here on Slack.](https://metaboat.slack.com/archives/C505ZNNH4/p1715098187144839).

I believe we should change the app counterpart to this a hard-coded value `Custom` as well.